oarepo_communities/ui/communities_components/templates/oarepo/invenio_communities/details/header.html

```diff
@@ -0,0 +1,17 @@
+{% extends "semantic-ui/invenio_communities/details/header.html" %}
+
+{%- block subheader_buttons %}
+    <div
+    class="sixteen wide mobile sixteen wide tablet three wide computer right aligned middle aligned column">
+    {%- if not community_use_jinja_header %}
+        {# TODO: Need to figure out what to do about the url as we dont have access 
+        to our input as templates are rendered on their end.
+         #}
+        <a href="/docs/_new?community={{ community.slug }}"
```

Comment:
For the time being, use settings and format?
```
<a href="{{ config.NEW_RECORD_PAGE.format(community_slug=community.slug) }}"
```

or

```
<a href="{{ config.RECORD_ROUTES['new'].format(community_slug=community.slug) }}"
```
(in sync with config.COMMUNITY_ROUTES used on frontpage.html)